src/mewbot/api/registry.py

```diff
@@ -18,6 +18,7 @@ class ComponentRegistry(abc.ABCMeta):
     """MetaType which reg"""
 
     registered: List[Type[Any]] = []
+    uuid: str
```

Comment:
I... am a little concerned that this works

Reply:
Honestly? So was I!

I wrote it, thought it would fail, then checked to see if the linters complained, and they did not.

Namespaces are a hell of a drug.